templates/support/index.html

```diff
@@ -12,7 +12,7 @@
     <div class="col-8">
       <h1>Ubuntu Advantage</h1>
       <h4>Security, support and managed open source for enterprises</h4>
-      <p>Future-proof the full stack, from the data centre to containers to the database, LMA, server and cloud-native applications with open source software support from Canonical.</p>
+      <p>Future-proof the full stack, from the data centre to containers to the database, <abbr title="Logging, Monitoring and Aggregation services">LMA</abbr>, server and cloud-native applications with open source software support from Canonical.</p>
```

Comment:
I always thought it was "Logging, monitoring and alerting"?

Reply:
@anthonydillon me too!  But https://ubuntu.com/legal/ubuntu-advantage-service-description says otherwise... @lechsandecki ?

Reply:
@campbieil do you know if the UASD is wrong and it should be 'Alerting' or 'Aggregation'?

Reply:
@anthonydillon based on comments in the issue I am leaving this as it is at least consistent inside Canonincal.